24H Study Room/Mocks/MockFireAuthService.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MockFireAuthService.swift
+//  24H Study Room
+//
+//  Created by Jia-Jiuan Tsai on 2022-09-05.
+//
+
+import Foundation
+
+class MockFireAuthService: FireAuthServicing {
+    func createUser(email: Email, password: Password, completion: @escaping (Result<UserCredential, Error>) -> Void) {
+        print("MockFireAuthService: trying to create a new user")
```

Comment:
The code to print things in the mock is likely not needed. A mock is typically only used in tests, where prints are not seen